eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -79,20 +82,14 @@ jobs:
         ServiceDirectory: "template"
         TestPipeline: ${{ parameters.TestPipeline }}
 
-    - pwsh: |
-        $toxenvvar = "whl,sdist,mindependency"
-        if ('$(System.TeamProject)' -eq 'internal') {
-          $toxenvvar = "whl,sdist,depends,latestdependency,mindependency,whl_no_aio"
-        }
-
-        # ensure that the variable is unset. if it isn't, use the value discovered there
-        if ('$(Run.ToxCustomEnvs)' -ne ('$' + '(Run.ToxCustomEnvs)'))
-        {
-          $toxenvvar = '$(Run.ToxCustomEnvs)'
-        }
-
-        echo "##vso[task.setvariable variable=toxenv]$toxenvvar"
-      displayName: "Set Tox Environment"
+    - task: PythonScript@0
```

Comment:
Out of curiosity, why the "@0"?

Reply:
It's the version of the job!  The way that you access native devops tasks is by the name + the version. [here is the index of native tasks](https://github.com/microsoft/azure-pipelines-tasks/tree/master/Tasks)

![image](https://user-images.githubusercontent.com/479566/127345416-02ece0a4-33a9-4778-80d8-b7279df03e51.png)